server/service/src/requisition/program_settings/supplier_program_settings/prepare.rs

```diff
@@ -72,7 +72,6 @@ pub(super) fn prepare(
         RequisitionsInPeriodRepository::new(&ctx.connection).query(filter)?;
 
     // Suppliers, which are visible in current store and have these program (this is determined by having program master list visible)
-    // TODO confirm if they are strictly stores, i.e. can't make internal program order (requisition) to a non store supplier
```

Comment:
Removed this comment because suppliers are already being filtered out by if they are a store in repo layer